UI/GameSettingsScreen.cpp

```diff
@@ -1273,6 +1278,11 @@ void GameSettingsScreen::dialogFinished(const Screen *dialog, DialogResult resul
 	}
 }
 
+void GameSettingsScreen::RecreateViews() {
+	oldSettingInfo_ = settingInfo_->GetText();
+	recreateViews_ = true;
```

Comment:
Why not call `UIScreen::RecreateViews()` instead?  We should keep that the sole interface and keep `recreateViews_` private.

-[Unknown]

Reply:
Ye good point :)